sdk/areas/areas.go

```diff
@@ -66,28 +66,29 @@ func (c *Client) Checker(ctx context.Context, check *health.CheckState) error {
 }
 
 // GetArea returns area information for a given area ID
-func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, areaID, acceptLang string) (areaDetails AreaDetails, err error) {
+func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, areaID, acceptLang string) (AreaDetails, error) {
+	var areaDetails AreaDetails
 	uri := fmt.Sprintf("%s/v1/areas/%s", c.hcCli.URL, areaID)
 	clientlog.Do(ctx, "retrieving area", service, uri)
 	resp, err := c.doGetWithAuthHeaders(ctx, userAuthToken, serviceAuthToken, collectionID, uri, nil, "", acceptLang)
 	if err != nil {
-		return
+		return areaDetails, nil
```

Comment:
Swallowing the error. If the err is not nil, then why are we returning a nil error?

Reply:
Good spot. Apologies I missed this whenI was fixing the lint issues in bulk.